packages/app/src/common/api/accounts.ts

```diff
@@ -0,0 +1,48 @@
+import type {
+  AddressBalanceResponse,
+  MempoolTransaction,
+  TransactionResults,
+} from '@blockstack/stacks-blockchain-api-types';
+
+import { fetchPendingTxs } from '@common/api/transactions';
+import { fetchFromSidecar } from '@common/api/fetch';
+
+export const fetchBalances = (apiServer: string) => async (
+  principal: string
+): Promise<AddressBalanceResponse> => {
+  const path = `/address/${principal}/balances`;
+  const res = await fetchFromSidecar(apiServer)(path);
+  return res.json();
+};
+
+export const fetchTransactions = (apiServer: string) => async (
```

Comment:
I like this currying pattern of passing apiserver first 👍🏼 

Keeps options open later, can do thinks like map bind a given server etc, good pattern to follow